### PR TITLE
Overlapping Button on Android Screen

### PR DESCRIPTION
**Pull request by piyushs131**
Issue Description:
After opening the heatmap and going to download the YAML file, I noticed that on the Android screen, the "Download YAML File" button and the "Reset implemented " button are overlapping. This makes it challenging to interact with both buttons effectively, as they interfere with each other functionality.

Steps to Reproduce:
1: Open the heatmap feature.
2: Proceed to download the YAML file.
3: Notice the overlapping of the "Download YAML File" and "Reset" buttons on the Android screen.

Expected Behavior:
The buttons should be displayed in a non-overlapping manner, ensuring that users can easily interact with both without any interference.

Screenshot 2024-03-10 185222

### CODE REVIEW

File: src/app/component/circular-heatmap/circular-heatmap.component.html [hunk 1]
```diff
@@ -269,22 +269,24 @@ <h2>Nothing to show</h2>
...
           </mat-card>
-        </div>
+      
+          <div class="button">
+            <button class="bt"
+            mat-raised-button
+            class="Download" 
+            (click) = "SaveEditedYAMLfile()"> 
+            Download Edited YAML file
+          </button>
```
Comment (wurstbrot) [inline src/app/component/circular-heatmap/circular-heatmap.component.html:279]: Please format correctly

File: angular.json [hunk 2]
```diff
@@ -116,6 +116,7 @@
   },
   "defaultProject": "DSOMM",
   "cli": {
+    "analytics": "c2f4e0c8-d08b-4ecc-b2c9-ba87bf702f30",
```
Comment (wurstbrot) [inline angular.json:119]: why is this needed?

**Comment by piyushs131** [comment]
Extra angular JSON file code has been generated following my recent update of the JSON file package in the codebase. 

Rest, I will promptly correct the format of the code and review the package.json code and also remove any unnecessary elements if they are not required.
